addons/mass_mailing/static/src/scss/mass_mailing.scss

```diff
@@ -221,5 +221,6 @@
 
     iframe {
         width: 100%;
+        min-height: 55vh;
```

Comment:
I assume it was also tested in a mobile environment?

Reply:
I had only tested in the browser with mobile size but I have now tested it on my smartphone and it works fine.